src/omero_cli_zarr.py

```diff
@@ -67,6 +70,21 @@ def _configure(self, parser):
 
     parser.set_defaults(func=self.export)
 
+
+  def _get_repo(self):
+    try:
+      if 'OMERODIR' in os.environ:
+        base_dir = Path(os.environ.get('OMERODIR'))
+      else:
+        self.ctx.die(1, 'OMERODIR env variable not set')
+      grid_dir = base_dir / "etc" / "grid"
+      cfg_xml = grid_dir / "config.xml"
+      config = ConfigXml(str(cfg_xml))
+      return config['omero.data.dir']
```

Comment:
The managed repo can be configured separately. `omero.clients.BaseClient` has a:

```
(prx, desc = getManagedRepository(description=True)
```
method that might be best.

Reply:
👍 Thanks. Yes, that'd be much easier too.